docs/html/topics/authentication.md

```diff
@@ -175,6 +177,12 @@ $ pip config set global.force-keyring true
 $ export PIP_FORCE_KEYRING=1
 ```
 
+Doing so will ensure that keyring is queried before contacting the server thus
+resulting in requests being authenticated. This can save a round-trip when you
+know beforehand that the server requires it. It can also prevent issues with
+Artifactory or other servers with overly strict security policies that blocks
+user (temporarily) on the first failed attempt.
+
 ```{warning}
```

Comment:
Please remove this warning. It is identical to the one on line 159, which is entirely my fault.

Reply:
Done. :-)